app/views/users/show.html.erb

```diff
@@ -1 +1,45 @@
-<h1>This is the show user section for testing</h1>
+<div class="container-sm" style="Margin-top: 20px;">
+  <div class="card mb-3" style="border: none;">
+    <div class="row g-0" style="gap: 2.5%;">
+      <div class="col-md-4" style="width: 70px;">
+        <img src ='<%= @user.photo %>' class="img-fluid rounded" alt="User Photo">
+      </div>
+      <div class="col-md-8">
+        <div class="card-body" style="border: 1px solid black; margin-top: 5px; width: 1195px;">
+          <h5 class="card-title" style="text-decoration: none"><%= @user.name %></h5>
+          <p class="card-text"><small class="text-muted">
+            Number of posts: <%= @user.posts_counter %></small>
+          </p>
+        </div>
```

Comment:
- [Optional] It will be nice if we ensure that our codebase is DRY(Don't Repeat Yourself). You'll see this piece of code through out your views. You can avoid this through partials usage :+1: